packages/samples/headless-ssr/README.md

```diff
@@ -0,0 +1,17 @@
+## About
+
+- This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+- It demonstrates usage of the `@coveo/headless/ssr` utils for Server-Side Rendering with headless.
+- Although NextJS is used to demonstrate usage for convenience, the utils are not specific to NextJS/React.
```

Comment:
One caveat with using React for this sample is that there's a chance customers may see it and follow it for their React implementation, not knowing that we also offer React utilities.

I think we could just add a note about it here once we add the React utils.